content/_code-samples/escrow/js/finish-escrow.js

```diff
@@ -0,0 +1,101 @@
+        for (var i = 0; i < response.result.account_objects.length; i++) {
+            if (response.result.account_objects[i].Account == wallet.address) {
+                outgoing.push(response.result.account_objects[i])
+            }
+        }
+        // Get current datetime
+        const date = new Date();
+
+        // Only append outgoing Escrows that have passed their FinishAfter time
+        var outgoing_finishable = []
+        for (var i = 0; i < outgoing.length; i++) {
+            const FinishAfter_date = new Date(xrpl.rippleTimeToISOTime(outgoing[i].FinishAfter));
+            if (FinishAfter_date.getTime() <= date.getTime()) {
+                outgoing_finishable.push(outgoing[i])
+            } else if (FinishAfter_date.getTime() >= date.getTime()) {
+                console.log(`Escrow ${outgoing[i].index} has not passed its FinishAfter date, it will pass it in ${FinishAfter_date}`)
+            }
+        }
+
+        if (outgoing_finishable.length != 0 ) {
+            console.log("\nOutgoing/Sent escrow(s) that have passed their FinishAfter time:")
+            for (var i = 0; i < outgoing_finishable.length; i++) {
+                console.log(`\n${i+1}. Index (ObjectID/keylet): ${outgoing_finishable[i].index}`)
+                console.log(` - Account: ${outgoing_finishable[i].Account})`)
+                console.log(` - Destination: ${outgoing_finishable[i].Destination}`)
+                console.log(` - Amount: ${outgoing_finishable[i].Amount} drops`)
+                console.log(` - PreviousTxnID: ${outgoing_finishable[i].PreviousTxnID}`)
+            }
+
+            console.log(`\nWe're going to finish Escrow ${outgoing_finishable[0].index} which was destined for ${outgoing_finishable[0].Destination} @ ${outgoing_finishable[0].Amount} drops...`)
+        
+            // Get the transaction's sequence, this transaction created this Escrow
+            const get_escrow_seq = await client.request({
+                "command": "tx",
+                "transaction": outgoing_finishable[0].PreviousTxnID
+            })
+
+            // Construct EscrowFinish transaction
+            const EscrowFinish_tx = await client.autofill({
+                "TransactionType": "EscrowFinish",
+                "Account": wallet.address,
+                "Owner": wallet.address,
+                "OfferSequence": get_escrow_seq.result.Sequence 
+            })
+
+            const EscrowFinish_tx_signed = wallet.sign(EscrowFinish_tx)
+
+            console.log("\n Transaction hash:", EscrowFinish_tx_signed.hash)
+
+            const EscrowFinish_tx_result = await client.submitAndWait(EscrowFinish_tx_signed.tx_blob)
+            
+            console.log("\n Submit result:", EscrowFinish_tx_result.result.meta.TransactionResult)
+            console.log("    Tx content:", EscrowFinish_tx_result)
+        } else {
+        console.log(`\nThere are no Escrows that have pass their FinishAfter due date on account ${wallet.address}`)
```

Comment:
nit: indent.  Github wont let me edit directly.

Reply:
Thanks for catching that ❤️